Dockerfile

```diff
@@ -31,11 +31,18 @@ COPY --chown=appuser:appuser . .
 RUN yarn build
 
 # =============================
-FROM nginx:1.19 as production
+FROM rhel8/nginx-116 as production
 # =============================
 
+USER root
+
+RUN chgrp -R 0 /usr/share/nginx/html && \
+    chmod -R g=u /usr/share/nginx/html
+
+USER 1001
+
 # Copy static build
-COPY --from=staticbuilder --chown=nginx:nginx /app/build /usr/share/nginx/html
+COPY --from=staticbuilder /app/build /usr/share/nginx/html
 
 # Copy nginx config
 COPY ./nginx/nginx.conf /etc/nginx/nginx.conf
```

Comment:
There seems to be some problem with either this Nginx, Docker, or environment configuration because the app does not respond correctly to other paths. For example /resource/tprek:8215